backend/molgenis-emx2/src/main/java/org/molgenis/emx2/ColumnType.java

```diff
@@ -120,4 +120,8 @@ public boolean isFile() {
   public boolean isHeading() {
     return HEADING.equals(getBaseType());
   }
+
+  public boolean isAtomicType() {
```

Comment:
why not use a class AtomicType instead of a function

Reply:
a never mind then , although this is starting to be a pretty complex enum ( with types and subtypes)

Reply:
could still use a interface , but then i looked at the usage and see this is only used in a test context, would consider removing the type test